charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -65,32 +117,68 @@ spec:
       containers:
         - name: csi-external-provisioner
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiProvisioner" . }}/csi-provisioner:v0.2.0"
+          image: "docker.io/openstorage/csi-provisioner:v1.6.1-1"
+          args:
+            - "--v=3"
+            - "--provisioner=pxd.portworx.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--enable-leader-election"
+            - "--leader-election-type=leases"
```

Comment:
these args will change based on the above image version. Same goes for snapshotter/snapshot-controller